README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
+  destinationHoldDuration: 3000
+})
+
+const json = packet.deserializeIlqpLiquidityRequest(binary)
+```
+
+#### QuoteLiquidityResponse
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityResponse({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  liquidityCurve: Buffer.alloc(16), // Must be a buffer of size (n * 16) bytes
```

Comment:
> For anyone other than five-bells-routing, liquidity curves should be an opaque object, i.e. you don't know what its internal representation is.

I don't think that's true. How about a client that requests a liquidity curve for a sending UI to avoid sending a quote request to the connector every time the user changes the amount?

> The current format (array of pairs of numbers) doesn't work because JavaScript numbers don't have 64-bit precision. So we're going to get a lot of precision related bugs unless we change the format.

Then why not have `ilp-packet` parse the numbers using a big integer library? If you're worried about other people wanting to use other libraries for manipulating uint64s, there could be an option to turn that off or return a string version. But I think the default behavior should be to parse the message fully.

For the ILP packet, returning the data as a buffer makes some amount of sense because that's meant for a different layer. For this quoting message I don't think it makes as much sense because there's no way to interpret this message without parsing the liquidity curve.